edb/edgeql-parser/src/keywords.rs

```diff
@@ -163,6 +163,7 @@ pub const CURRENT_RESERVED_KEYWORDS: &[&str] = &[
     "like",
     "limit",
     "module",
+    "never",
```

Comment:
Perhaps we should add it as an abstract final `std::never` type, but that would require some hacks.

Reply:
OTOH `never` isn't exactly a popular property/link/function/parameter name.